### PR TITLE
Mentor

### PR DESCRIPTION
**Pull request by Stephanie-A**
Deploy mentor site

### CODE REVIEW

**Comment by 1ntEgr8** [comment]
going to wait for @mohandodda0 to integrate cms

File: src/components/est-nav.js [hunk 1]
```diff
@@ -118,19 +119,21 @@ class EstNav extends HTMLElement {
     });
     const currentLink = this._targets[this.sectionIndex].link;
     const newLink = this._targets[i].link;
-
+    
+      console.log(currentLink);
```
Comment (1ntEgr8) [inline src/components/est-nav.js]: remove these console logs

**Review by 1ntEgr8 — APPROVED** [review]
lgtm; remove the console logs and then merge!

File: src/components/est-nav.js (hunk 1 above)
Comment (mohandodda0) [reply src/components/est-nav.js]: Ok done. Am merging.

**Pull request merged**